server/akka-http-server/src/test/scala/sttp/tapir/server/akkahttp/AkkaHttpCreateServerTest.scala

```diff
@@ -43,6 +53,34 @@ class AkkaHttpCreateServerTest extends TestSuite {
               basicRequest.get(uri"http://localhost:$port/api/test/directive").send(backend).map(_.body shouldBe Right("ok"))
             }
             .unsafeRunSync()
+        },
+        Test("Send and receive SSE") {
+          implicit val ec = scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
you can use `actorSystem.dispatcher` which is also used above

Reply:
Missed that one, replaced